src/hooks/use-last-seen-version.tsx

```diff
@@ -0,0 +1,47 @@
+import { useState, useEffect, useCallback } from 'react';
+import { getIpcApi } from 'src/lib/get-ipc-api';
+
+interface UseLastSeenVersion {
+	lastSeenVersion: string | undefined;
+	isNewVersion: boolean;
+	updateLastSeenVersion: () => Promise< void >;
+	isLoading: boolean;
+}
+
+export function useLastSeenVersion(): UseLastSeenVersion {
```

Comment:
What you considered another approach for the empty `lastSeenVersion` on the userData?
I think that if we returned something by default for the `lastSeenVersion` when no version is found? If we did the logic on this hook would be easier to understand, and we would avoid having the `isLoading` state.
What do you think?

Reply:
That's a great idea. Thanks. I have made the change in https://github.com/Automattic/studio/pull/1110/commits/4f8423ad1b16fc2d23e6c34dd44523dd08084440.